docs: fix linter errors for updating.md (#47837)
angular/angular: aio/content/guide/updating.md

This document contains information and resources to help you keep your Angular applications and libraries up-to-date.

> For information about the versioning policy and practices &mdash;including support and deprecation practices, as well as the release schedule&mdash; see [Angular versioning and releases](guide/releases "Angular versioning and releases").

Reviewer: This is really the Angular team's policy, so instead of "our," saying that outright would be the clearest.
```suggestion
For information about the Angular team's versioning policy and practices, their support and deprecation practices, and the release schedule, see [Angular versioning and releases](guide/releases "Angular versioning and releases").
```

## Learning about new features

What's new? What's changed? The most important things you need to know are shared on the Angular blog in [release announcements]( https://blog.angular.io/tagged/release%20notes "Angular blog - release announcements").

To review a complete list of changes, organized by version, see the [Angular change log](https://github.com/angular/angular/blob/main/CHANGELOG.md "Angular change log").

<a id="checking-version-app"></a>

## Checking your version of Angular

> To see your application's version of Angular: from within your project directory, use the `ng version` command.

Reviewer: Let's liberate the list that's hiding in the sentence:
```suggestion
To see your application's version of Angular: 

1.  Navigate to your Angular application's project directory.
1.  Run `ng version`.
```

The most recent stable released version of Angular appears in the [Angular documentation](docs "Angular documentation") at the bottom of the left side navigation.
For example, `stable (v13.0.3)`.

You can also find the most current version of Angular by using the [`ng update`](cli/update) command.

> By default, [`ng update`](cli/update)(without arguments) lists the updates that are available to you.

Reviewer: Does this need a space after the command?
```suggestion
By default, [`ng update`](cli/update) (without arguments) lists the updates that are available to you.
```

<a id="updating"></a>

## Updating your environment and apps

> To make updating uncomplicated, complete instructions are provided in the interactive [Angular Update Guide](https://update.angular.io/ "Angular Update Guide").

Reviewer: Could this be change to active voice, such as:
```suggestion
To make updating easier, the interactive [Angular Update Guide](https://update.angular.io/ "Angular Update Guide") provides specific instructions for your version of Angular.
```
